packages/components/src/components/bal-hint/test/bal-hint.visual.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <script type="module" src="/build/design-system-components.esm.js"></script>
+    <script nomodule src="/build/design-system-components.js"></script>
+  </head>
+
+  <body>
```

Comment:
can u add this before the script in the head

```html
    <style>
      .bal-body {
        visibility: hidden;
      }
    </style>
```

and set `.bal-body` to the body element

Reply:
yes, sure